src/config.ts

```diff
@@ -22,6 +22,9 @@ export const hideClassify = core.getInput("hide_classify", {
   required: true
 }) as ReportedContentClassifiers
 export const deleteOldComment = core.getBooleanInput("delete", {required: true})
+export const onlyCreateComment = core.getBooleanInput("only_create", {
+  required: false
```

Comment:
It's a little hard to figure out, but since we've specified a default in your config, it should always have a value at this point.
```suggestion
  required: true
```

Reply:
ah thanks, I was wondering what was going on here